### ISSUE TITLE
Freeday Particle Problem Again?)

### ISSUE THREAD

**Issue by MrPanica** [Scags/TF2-Jailbreak-Redux#128]
Hello!

I already wrote about this bug in a previous version of the plugin (https://github.com/Scags/TF2-Jailbreak-Redux/issues/92).
But since that time I have become more experienced in terms of coding.
And I seem to have found the problem.
Your plugin has a client property "iFreedayParticle" which is responsible for the freeday particle.

But the problem is that the freeday is issued at the "OnPlayerSpawn" event, which fires before the "OnPreRoundStart" event, where the "iFreedayParticle" property is cleared.

In other words, the property is cleared immediately after the freeday is issued to the player, which makes it impossible to remove the particle.

By the way, it was because of this that I could remove the Freeday particles when issuing it with the "sm_givefreeday" command.

**Comment by Scags** [comment]
Good catch. I'll fix this when I finally get around to updating the plugin again.
